pkg/config/config_template.yaml

```diff
@@ -1256,11 +1256,23 @@ api_key:
   # blacklist_patterns:
   #   - <REGEX>
 
-  ## @param queue_size - integer - optional - default: 20
-  ## @env DD_PROCESS_CONFIG_QUEUE_SIZE - integer - optional - default: 20
+  ## @param queue_size - integer - optional - default: 256
+  ## @env DD_PROCESS_CONFIG_QUEUE_SIZE - integer - optional - default: 256
   ## How many check results to buffer in memory when POST fails.
   #
-  # queue_size: 20
+  # queue_size: 256
+
+  ## @param process_queue_bytes - integer - optional - default: 60000000
+  ## @env DD_PROCESS_CONFIG_PROCESS_QUEUE_BYTES - integer - optional - default: 60000000
+  ## How much check data (in bytes) to buffer in memory when POST fails.
```

Comment:
nit: 
```suggestion
  ## The amount of data (in bytes) to buffer in memory when a POST fails.
```

Reply:
Thanks for the suggestion! I've just updated it!